include/miopen/miopen.h

```diff
@@ -592,7 +592,7 @@ MIOPEN_EXPORT miopenStatus_t miopenCreateTensorDescriptor(miopenTensorDescriptor
  *
  * Interface for setting 4-D tensor shape. MIOpen currently only implements NCHW layout.
  *
- * @param tensorDesc Tensor descriptor type (output)
+ * @param tensorDesc Tensor descriptor (input)
```

Comment:
Formally, this is an input parameter, but given that this call modifies the internal state of the existing descriptor, the parameter is actually input/output.

Reply:
The description has been fixed

Reply:
[Notice] @averinevg In trivial cases, please mark fixed review comments as _resolved_ thus hiding them (I can't do that)